packages/graphql-docs-generator/README.md

```diff
@@ -1,16 +1,36 @@
 # GraphQL Docs generator
-GraphQL document generator takes a schema and generates all possible statements(queries, mutations and subscription) on that schema. This can act as a starting point for the users who are new to GraphQL
+GraphQL document generator takes a schema and generates all possible statements(queries, mutations and subscription) on that schema. This can act as a starting point for the users who are new to GraphQL.
 
 ## Installation and execution
+#### Executing from the command line:
 ```
 $ npm install -g @aws-amplify/graphql-docs-generator
-$ graphql-docs-generator --schema test-data/schema.json --output all-operations.graphql --language graphql
-``` 
-
-## Todo:
-- [x] Add support for subscriptions
-- [x] Add type information
-- [X] Add unit tests
-- [ ] Support generating operation by downloading schema from a GraphQL server
-- [x] Support generation of separate files (queries.graphql, mutations.graphql and subscriptions.graphql)
-- [ ] Support de-duping of fragments (non inline fragments)
+
+$ graphql-docs-generator --schema 'my-graphql-sdl-schema'
+
+$ graphql-docs-generator --schema 'my-graphql-introspection-schema'  --language 'graphql' --maxDepth 2  --isSDLSchema false
```

Comment:
The `--schema` here is changed from file path to the schema string, which is not as easy as the previous one. Is this command intended for the usage without installing amplify cli?

Reply:
Yup, that's right. I have another task to disable standalone CLI invocation of this package given this updated API makes sense for programmatic invocations, so this is going to go away soon.